### DISCUSSION TITLE
How to get a pole figure with the unit is mud（multiple of uniform density）？

### DISCUSSION THREAD

**Discussion by YJoyce** [mtex-toolbox/mtex#2004] [Ask Anything]
Hello, everyone!

There's a question that's been bothering me for a long time. When I plot pole figures in OIM, I can get scatter plots and polar density plots (unit is MUD).

When I try to plot pole figures in MTEX, I realize that there are two ways to get contour pole plots (from orientation or from ODF). What is the difference between them? Which method agrees with the results from OIM?

Finally, I got another concept "MRD (Multiples of Random Distribution)" from our community, is it the same as MUD?

Thank you all very much for your help and support! I would appreciate any insights and advice you can share with me on this topic and I look forward to hearing from you!

**Comment by kilir** [comment]
Hi,
if you do soemthing like `plotPDF(ori,h,'contourf')` density estimation is done on the sphere, individually for each pole figure, if you do `plotPDF(odf,h)`, density estiamtion has happened in SO(3). However, what you get in OIM should be written in their documentation.

MRD; maybe you can explain how your community defines it, such that it should differ from MUD?

Cheers,
Rüdiger

**Reply by YJoyce** [reply]
Hi Rüdiger,

Thanks for your reply, I know the difference between plot pole figures from orientation and from ODF.

From the website https://mtex-toolbox.github.io/ODFTheory.html, the resulting value of ODF needs to be interpreted as multiple of random distribution (mrd). However, I usually come across units of mud when analyzing EBSD data, which gives me some confusion and I would like to know the difference between these units.

Best,
Jing

**Reply by kilir** [reply]
In that case, it's the same.
Cheers,R
Rüdiger
